### ISSUE TITLE
Revamp "Empty project" template

### ISSUE THREAD

**Issue by samajammin** [SuperblocksHQ/ethereum-studio#269]
## Summary

Let's refresh the README of the "Empty project" template in the same fashion as #234 #241 & #246.

## Motivation

Current instructions & links are outdated.

## Describe alternatives you've considered

Should we consider removing this entirely? Do we have proof that anyone has successfully created a project from this blank slate template? What else would need to be added/changed to actually allow someone to build their own dapp with this template? e.g. should we include instructions on how to add a frontend form field? Or instructions on using the "Interact" panel? Should we make it clear this template is for advanced users, who already know how to code a smart contract & a frontend?

## Additional context
None

**Comment by javier-tarazaga** [comment]
Good point. Once we have the onboarding of the tool in place, I think the next thing we need to do is to actually remove all references to instructions on how to use the tool in the readmes. This makes it really cumbersome and adds a ton of work when ever we are changing anything in the UI. 

In regards of the empty template, yes we have some number and in fact it is being used. For now it is the 3 most used example, after Hello World (1st) and Token (2nd)
